src/main/scala/io/iohk/ethereum/vm/DataWord.scala

```diff
@@ -98,4 +110,9 @@ class DataWord private (private val n: BigInt) extends Ordered[DataWord] {
     f"DataWord(0x$n%02x)" //would be even better to add a leading zero if odd number of digits
 
   def toBigInt: BigInt = n
+
+  /**
+    * @return Size in bytes excluding the leading 0 bytes
+    */
+  def byteSize: Int = if (isZero) 0 else (n.bitLength - 1) / 8 + 1
```

Comment:
If you're still not convinced, here's the proof:

```
def:
  n.bitLength = floor(log_2(n))  + 1 // not going to prove that, consider some examples

then: 
  n.bitLength - 1 = floor(log_2(n))
  1 + floor((n.bitLength - 1) / 8) = 1 + floor(log_2(n) / 8) = 1 + floor(log_2(n) / log_2(256))

thus:
  1 + floor((n.bitLength - 1) / 8) = 1 + floor(log_256(n))

QED
```

Is that convincing enough? 😁